docs/integrations/task-complete.md

```diff
@@ -0,0 +1,10 @@
+---
+id: taskComplete 
+title: taskComplete 
+---
+You can use this command to reopen a closed incident. For example, you can use the reopenInvestigation command to add an action button in the incident layout, that can reopen a closed incident and take a specific action, such as rerunning the incident's playbook. You can also include the reopenInvestigation command in a loop in a script to reopen multiple incidents. 
+
+| Argument         | Description           | 
+| ------------- |-------------|   
+| id     | Which incident to reopen. If no incident ID is provided, the current incident is reopened.      |   
+   
```

Comment:
Looks like the wrong description (the one for `reopenInvestigation`) got copied here.